panther_driver/src/driver_node.py

```diff
@@ -225,24 +253,31 @@ def _driver_state_timer_cb(self, *args) -> None:
             self._driver_state_msg.rear.temperature
         ] = self._panther_can.query_driver_temperature_data()
 
-        self._roboteq_fault_flags = list(self._panther_can.query_fault_flags())
-        self._roboteq_script_flags = list(self._panther_can.query_script_flags())
-        self._roboteq_runtime_flags = list(self._panther_can.query_runtime_stat_flag())
-
-        self._driver_state_msg.front.fault_flag = self._decode_fault_flag(self._roboteq_fault_flags[0])
-        self._driver_state_msg.rear.fault_flag = self._decode_fault_flag(self._roboteq_fault_flags[1])
+        [
+            self._driver_state_msg.front.fault_flag, 
+            self._driver_state_msg.rear.fault_flag,
+        ] = [
+            self._decode_flag(msg_obj=FaultFlag(), msg_fields_list=self._faults_list, flag_val=flag_val)
```

Comment:
```suggestion
            self._decode_flag(FaultFlag(), self._faults_list, flag_val)
```
Wouldn't it be simpler?